.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 npm-debug.log
+.idea
```

Comment:
This shouldn't be here.
You can create a global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
or add it to your local repo ignores: `.git/info/exclude`